references/segmentation/coco_utils.py

```diff
@@ -68,11 +68,6 @@ def _has_valid_annotation(anno):
         # if more than 1k pixels occupied in the image
         return sum(obj["area"] for obj in anno) > 1000
 
-    if not isinstance(dataset, torchvision.datasets.CocoDetection):
-        raise TypeError(
-            f"This function expects dataset of type torchvision.datasets.CocoDetection, instead  got {type(dataset)}"
-        )
```

Comment:
Do we remove this, because it is an internal function and there should be never anything else?

Reply:
Yeah I don't think this helps a ton and until https://github.com/pytorch/vision/pull/7239 is merged it makes things easier